quotecache/quotecache.go

```diff
@@ -75,7 +82,7 @@ func updateQuoteCache(userID, stock string) error {
 		return err
 	}
 
-	cache[stock] = quote
+	QuoteCache[stock] = map[string]Quote{userID: quote}
```

Comment:
I don't think so, since we want to make a user's ID to their valid quote

```
"AAPL": {"John": John'sQuoteInstance}
```

VS

```
"AAPL": {"John": "John"}
```

Go is smart enough to determine that userID is a variable, and thus it will auto expand it. So the cache will actually look like the first one instead of

```
"AAPL": {"userID": John'sQuoteInstance}
```

It's essentially doing [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Computed_property_names) implicitly.